releases/v0.22.md

```diff
@@ -3,9 +3,9 @@
 This document contains all release notes pertaining to the `v0.22.x` releases of
 Vela.
 
-## v0.22.0
+## v0.22.1
```

Comment:
typically we have an extra heading above this one, see https://github.com/go-vela/community/blob/main/releases/v0.20.md#v0201 as an example